Dockerfile

```
@@ -0,0 +1,12 @@
+FROM node:18-slim
+
+LABEL maintainer="engineering@dhiway.com"
+
+WORKDIR /demo
+COPY . /demo
+
+RUN npm install
+
+EXPOSE 30333 9933 9944 9615
+
+ENTRYPOINT [ "npm" , "run" , "demo" ]
```

can we just provide like below and ask the users to provide third option in command line? 
```suggestion
ENTRYPOINT [ "npm" , "run" ]
```

---

docker run command is updated to `docker run <image-name> demo`